packages/sdk/src/fetcher/subgraph/index.ts

```diff
@@ -126,15 +132,17 @@ const mapRawKPIToken = async (
         description = rawKPIToken.description.description;
         tags = rawKPIToken.description.tags;
     }
+
     return new KPIToken(
         chainId,
         getAddress(rawKPIToken.rawAddress),
         getAddress(rawKPIToken.rawOwner),
         await mapRawTemplate(ipfsGatewayURL, rawKPIToken.template),
+
         await Promise.all(
-            rawKPIToken.oracles.map(async (rawOracle) => {
-                return mapRawOracle(chainId, ipfsGatewayURL, rawOracle);
-            })
+            rawKPIToken.oracles.map(async (rawOracle) =>
```

Comment:
I guess we can remove `async` here since we just return the promises, without `waiting` on each one, right?

Reply:
I'm not sure. but will check

Reply:
You are right we can remove `async`. Fixed. Thanks!